src/components/LoginPage.js

```diff
@@ -82,10 +82,11 @@ export class LoginPage extends React.Component {
             <Button type="submit" value="login" variant="raised" color="primary">
               Login
             </Button>
-
             {isLoginPending && <Snackbar open={true} autoHideDuration={6000} message={<span>Logging you in...</span>} />}
             {/* TODO: find a way to make this conditional on what type of user you are */}
-            {isLoginSuccess && <Redirect to="/dashboard" />}
+            {isLoginSuccess && userPath != null && <Redirect to={userPath} />}
+            {/* {isLoginSuccess && (userType == "LM") && <Redirect to="/input" />} */}
+            {/* {userType == "QA" && <Redirect to="/dashboard" />} */}
```

Comment:
do we still need the commented out code here eventually?

Reply:
Nah we should be fine without them, just forgot to delete them.